examples/java/spring-boot/src/test/java/uk/gov/api/springboot/infrastructure/config/CorrelationIdFilterTest.java

```diff
@@ -31,6 +31,7 @@
   @Mock private ErrorResponseDecorator errorResponseDecorator;
   @Mock private MdcFacade mdcFacade;
   @InjectMocks private CorrelationIdFilter filter;
+  private final String correlationId = "93094CAB-21D7-43EC-97E9-566573544781";
```

Comment:
could even be made a `static final` as it's not needing to be shared per test?

Also, maybe name change to `EXAMPLE_VALID_CORRELATION_ID`?